client/gutenberg/extensions/markdown/editor.js

```diff
@@ -56,3 +57,5 @@ registerBlockType( 'jetpack/markdown', {
 
 	save,
 } );
+
+MarkdownBlock.requireModules( [ 'markdown' ] ).register();
```

Comment:
Could we have a simpler syntax here? I'm thinking that the module name will always be the same like the block type, without the prefix. Will we have a case where the module name will be different than the block name?